packages/i18n/data/core.json

```diff
@@ -36,30 +36,6 @@
   "ErrorApologizer.title": "Sorry! An unexpected error occured.",
   "FailedAuthentication.paragraph1": "Please try again and contact your system administrator if you have any further questions.",
   "FailedAuthentication.title": "We are unable to authorize you to use the Merchant Center.",
-  "Login.sso.caseSensitive": "This field is case sensitive",
-  "Login.sso.organization": "Enter your Organization's name to sign in",
-  "Login.sso.organizationGeneralError": "Organization name is not found",
-  "Login.sso.organizationGeneralErrorSecondary": "Either your Organization is not using SSO, is misspelled or does not exist. Try re-entering your Organization and contact your system administrator if you have any further questions.",
-  "Login.sso.proceed": "Proceed to sign in",
-  "Login.sso.title": "Sign in to your Organization",
-  "LoginForm.email": "Email",
-  "LoginForm.forgotPassword": "Forgot password?",
-  "LoginForm.forgotPasswordTitle": "We are redirecting you to the Forgot Password form in {countdown} seconds.",
-  "LoginForm.InvalidCredentials": "Sorry, but the login details entered below are not in our system. Please enter your correct email address and password below.",
-  "LoginForm.invalidEmail": "Please enter a valid Email address.",
-  "LoginForm.logout": "We have logged you out of the Merchant Center. If you would like to log back in, please fill out the form below.",
-  "LoginForm.noProjectsLink": "set up a project in the current Merchant Center",
-  "LoginForm.noProjectsTitle": "Sorry, but you do not have a project associated to your account and we are therefore unable to log you in. Please {link} before trying again.",
-  "LoginForm.password": "Password",
-  "LoginForm.signin": "Sign in",
-  "LoginForm.title": "Sign in to your account",
-  "LoginForm.unauthorized": "Your session has expired. Please sign in again to continue using the Merchant Center.",
-  "LoginForm.validating": "Validating...",
-  "LoginLocked.helpDeskLink": "Help Desk",
-  "LoginLocked.paragraph1": "For security reasons, your account has been locked because of too many failed log-in attempts. You can {resetPasswordLink} or contact the commercetools {helpDeskLink}",
-  "LoginLocked.resetPasswordLink": "reset your password",
-  "LoginLocked.title": "Account Locked",
-  "LoginPage.privacyPolicy": "Privacy Policy",
```

Comment:
Let’s also remove the translations for this

Reply:
2168293